tools/its.py

```diff
@@ -21,7 +22,9 @@
 
 from tools.clog import CLogger
 from tools.kontaktdaten import decode_wochentag, validate_kontakt, validate_zeitrahmen
-from tools.utils import retry_on_failure, desktop_notification
+from tools.utils import retry_on_failure, desktop_notification, update_available
+from pathlib import Path
+from urllib.request import urlopen
```

Comment:
```suggestion
```

Reply:
Kann raus - hatte davor die Prüfung in versch. Files bevor ich es als funktion in utils umgesetzt habe